test/foundry/new/FuzzEngine.t.sol

```diff
@@ -17,6 +17,7 @@ import {
     HashValidationZoneOfferer
 } from "../../../contracts/test/HashValidationZoneOfferer.sol";
 import { AdvancedOrder, FuzzHelpers } from "./helpers/FuzzHelpers.sol";
+import { CheckHelpers } from "./helpers/FuzzSetup.sol";
```

Comment:
Might make sense to move this to `FuzzEngineLib`, or `FuzzTestContextLib`, what do you think?

Reply:
Yeah, sg.  FuzzEngineLib seems like he right play based on gut feel.  Happy to reshuffle as necessary.  Will update and then merge into your branch